src/pages/iou/request/step/IOURequestStepScan/index.native.js

```diff
@@ -63,6 +65,7 @@ function IOURequestStepScan({
     const device = useCameraDevice('back', {
         physicalDevices: ['wide-angle-camera'],
     });
+    const [user] = useOnyx(ONYXKEYS.USER)
```

Comment:
should I use a selector for only `isMutedAllSounds` here, or is this fine?

Reply:
I dont think we can use useOnyx yet, please use selector

Reply:
@mountiny what is the reason we shouldn't use `useOnyx yet?